src/features/ui/FormInputs/FormInputs.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC, useState } from 'react';
```

Comment:
Is `features/ui` the appropriate place for this component? I think this would probably be better in `features/transfer-ownership`, unless we can identify this exact `input + button` pattern in other places.

Reply:
Oh yeah, it could make sense to put it in a shared folder if it's in those modals too! I think it could be handy to move this to zUI, but for now we can keep it in here.